diff --git a/steampipe_alchemy/models/aws_iam_user.py b/steampipe_alchemy/models/aws_iam_user.py
index 642f727..15eb37d 100644
--- a/steampipe_alchemy/models/aws_iam_user.py
+++ b/steampipe_alchemy/models/aws_iam_user.py
@@ -1,7 +1,10 @@
+from typing import List
+
 from sqlalchemy import Column
 from sqlalchemy.types import JSON, Text, Boolean, TIMESTAMP, BigInteger
 from sqlalchemy.dialects import postgresql as psql
 from steampipe_alchemy.mixins import FormatMixins
+from steampipe_alchemy.types.policy import Policy
 
 from steampipe_alchemy import Base
 
@@ -19,7 +22,7 @@ class AwsIamUser(Base, FormatMixins):
     mfa_devices = Column('mfa_devices', JSON, nullable=True)
     groups = Column('groups', JSON, nullable=True)
     inline_policies = Column('inline_policies', JSON, nullable=True)
-    inline_policies_std = Column('inline_policies_std', JSON, nullable=True)
+    inline_policies_std: List[Policy] = Column('inline_policies_std', JSON, nullable=True)
     attached_policy_arns = Column('attached_policy_arns', JSON, nullable=True)
     tags_src = Column('tags_src', JSON, nullable=True)
     tags = Column('tags', JSON, nullable=True)
@@ -27,4 +30,4 @@ class AwsIamUser(Base, FormatMixins):
     akas = Column('akas', JSON, nullable=True)
     partition = Column('partition', Text, nullable=True)
     region = Column('region', Text, nullable=True)
-    account_id = Column('account_id', Text, nullable=True)
\ No newline at end of file
+    account_id = Column('account_id', Text, nullable=True)
